test/content-negotiation-test.js

```diff
@@ -0,0 +1,61 @@
+var test = require('tape');
+var Hapi = require('hapi');
+var server = new Hapi.Server();
+var routes = require('../routes.js');
+
+server.connection({
+  port: process.env.TESTPORT || '8080'
+});
+
+server.route(routes);
+
+server.register(require('hapi-negotiator'), function (err) {
+  if (err) {
+    console.error('Failed to load plugin:', err);
+  }
+});
+
+test('Content Negotiation', function (t) {
```

Comment:
It's a good idea to always `t.plan` as the first thing you do in the test - this helps prevent false positives when assertions in conditionals or async code you expect to be run are (for whatever reason) not!